### DISCUSSION TITLE
Suggestion: auto convert dynamic pics from this website

### DISCUSSION THREAD

**Discussion by Violettx** [t1m0thyj/WinDynamicDesktop#472] [Ideas]
https://www.sony.net/united/clock/

Photos on this website absolutely fit this win dynamic desktop. it's a Sony camera showoff site recording the whole day of beautiful scenes all over the world. 

I'm wondering if we can use script to convert each photo to the dynamic desktop we need. 

I do not know about the copyright, so if we do it, must proceed with caution. (but as Apple original wallpaper is ok, so is this I guess)

**Comment by t1m0thyj** [comment]
These photos are indeed beautiful and look like a great fit for WDD. Do you know if there's a way to easily download a whole set of images? I only seem to be able to download one frame at a time.

**Comment by Violettx** [comment]
https://chrome.google.com/webstore/detail/imageassistant-batch-imag/dbjbempljhcmhlfpfacalomonjpalpko
I use ImageAssistant to download all the pictures. It can only download for 1080p tho. 
<img width="1412" alt="截屏2021-09-24 11 03 13" src="https://user-images.githubusercontent.com/29793911/134611789-9bcaf44f-863f-4513-8f48-ea634aa23b6a.png">

definitely know python script can automatically download for like 4k, not quite sure how to write a script lol

**Comment by ghost** [comment]
1080p is bad, we need 5k+ images!
